<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Brian Muchmore" />
    <meta name="date" content="2020-03-23" />
    <title>Brian Muchmore’s resume</title>

    
        <link href="resume_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
    <link href="resume_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
    <link href="resume_files/paged-0.9/css/resume.css" rel="stylesheet" />
    <script src="resume_files/paged-0.9/js/config.js"></script>
    <script src="resume_files/paged-0.9/js/paged.js"></script>
    <script src="resume_files/paged-0.9/js/hooks.js"></script>
    
    
  </head>
  <body>

<style type="text/css">
.pagedjs_page:not(:first-of-type) {
  --sidebar-width: 0rem;
  --sidebar-background-color: #ffffff;
  --main-width: calc(var(--content-width) - var(--sidebar-width));
  --decorator-horizontal-margin: 0.2in;
}
.break-after-me {
  break-after: page;
}
.spoiler {
  visibility: hidden;
}
</style>
<div id="aside" class="section level1">
<h1>Aside</h1>
<p><img src="https://brianmuchmore.com/authors/admin/avatar_hu78316369a6591e23ad8ee619d7773d6e_13319_250x250_fill_q90_lanczos_center.jpg" alt="Brian Muchmore" style="width:40.0%" /></p>
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto:bmuchmore@gmail.com" class="email">bmuchmore@gmail.com</a></li>
<li><i class="fa fa-envelope"></i> <a href="mailto:brian.muchmore@med.uvm.edu" class="email">brian.muchmore@med.uvm.edu</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/bmuchmore">github.com/bmuchmore</a></li>
<li><i class="fa fa-phone"></i> 240-408-6061</li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><p>10,000+ hours of laboratory bench work experience.</p></li>
<li><p>10,000+ hours of data science experience with working proficiency in R, Bash, and Python.</p></li>
<li><p>Professional proficiency using Docker and Guacamole.</p></li>
</ul>
</div>
<div id="disclaimer" class="section level2">
<h2>Disclaimer</h2>
<p>Last updated on 2020-03-23.</p>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Brian Muchmore</h2>
<div id="current-md-candidate-at-the-university-of-vermonts-larner-college-of-medicine." class="section level3">
<h3>Current MD candidate at the University of Vermont’s Larner College of Medicine.</h3>
</div>
</div>
<div id="education" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education</h2>
<div id="bates-college" class="section level3">
<h3>Bates College</h3>
<p>BA in Russian Studies</p>
<p>Lewiston, Maine</p>
<p>2002-2005</p>
<p>Graduated after six semesters of classes.</p>
</div>
<div id="nevskii-institute-of-language-and-culture" class="section level3">
<h3>Nevskii Institute of Language and Culture</h3>
<p>N/A</p>
<p>St. Petersburg, Russia</p>
<p>2004</p>
<p>Course work in Russian language, economics and history while living with a Russian family.</p>
</div>
<div id="odessa-language-study-center" class="section level3">
<h3>Odessa Language Study Center</h3>
<p>N/A</p>
<p>Odessa, Ukraine</p>
<p>2005</p>
<p>The Institute is a full EAQUALS member and evaluated my Russian language level as C1.</p>
</div>
<div id="university-of-maryland" class="section level3">
<h3>University of Maryland</h3>
<p>BSc.</p>
<p>College Park, Maryland</p>
<p>2009-2012</p>
<p>Completed the core requirements of a BSc and took various science electives while working full-time at the NIH.</p>
</div>
<div id="the-university-of-vermonts-larner-college-of-medicine" class="section level3">
<h3>The University of Vermont’s Larner College of Medicine</h3>
<p>MD candidate.</p>
<p>Burlington, Vermont</p>
<p>2018-2021</p>
<p>Worked full-time as a Bioinformatician for the first 18 months of school.</p>
</div>
</div>
<div id="research-experience" class="section level2" data-icon="laptop">
<h2>Research Experience</h2>
<div id="summer-fellowship" class="section level3">
<h3>Summer Fellowship</h3>
<p>Laboratory of Structural Biology; Dr. Alasdair Steven</p>
<p>National Institute of Arthritis and Musculoskeletal and Skin Diseases, NIH</p>
<p>2009</p>
<ul>
<li>Cloned mutants of the fungal prion protein Het-s for analysis of structural motifs with electron microscopy.</li>
</ul>
</div>
<div id="cancer-research-training-award-fellow" class="section level3">
<h3>Cancer Research Training Award Fellow</h3>
<p>Laboratory of Translational Genomics; Dr. Ludmila Prokunina-Olsson</p>
<p>Division of Cancer Epidemiology and Genetics, National Cancer Institute, NIH</p>
<p>2009-2012</p>
<ul>
<li>Functional investigation of GWAS cancer signals associated with HCV-related traits and bladder cancer.</li>
</ul>
</div>
<div id="research-scientist" class="section level3">
<h3>Research Scientist</h3>
<p>Bangkok Center of Excellence in Clinical Virology; Dr. Yong Poovorawan</p>
<p>Faculty of Medicine, Chulalongkorn University</p>
<p>2013-2014</p>
<ul>
<li>Evolutionary analysis of SNPs and INDELs using various bioinformatics techniques and publicly available data sets such as ancient genomes and 1000 genomes data.</li>
</ul>
</div>
<div id="research-scientist-1" class="section level3">
<h3>Research Scientist</h3>
<p>Genetics of Complex Diseases Group; Dr. Marta Alarcón Riquelme</p>
<p>Pfizer-University of Granada-Junta de Andalucía Centre for Genomics and Oncological Research (GENYO)</p>
<p>2015-2018</p>
<ul>
<li>Part of a multi-national and multi-institutional project led by Dr. Marta Alarcón Riquelme to find clinically useful biomarkers for systemic autoimmune diseases (www.precisesads.eu). Also one of the leads of Spain’s first CyTOF/Helios core facility.</li>
</ul>
<p class="spoiler">
Push Text Down
</p>
</div>
</div>
<div id="professional-experience" class="section level2" data-icon="suitcase">
<h2>Professional Experience</h2>
<div id="head-of-reporting-and-recurrent-consultant" class="section level3">
<h3>Head of Reporting and Recurrent Consultant</h3>
<p>CodoniXnotes EHR Software</p>
<p>Codonix</p>
<p>2014-Current</p>
<div class="concise">
<ul>
<li>Data mining of Codonix’s terabyte-plus of patient data.</li>
<li>Implemented a sophisticated real-time reporting solution using Docker and R (Shiny).</li>
<li>Created a combined Guacamole-Docker application to serve Codonix’s Java applet over HTML5.</li>
</ul>
</div>
<p class="spoiler">
Push Text Down
</p>
</div>
</div>
<div id="publications" class="section level2" data-icon="file">
<h2>Publications</h2>
<div id="from-noncoding-variant-to-phenotype-via-sort1-at-the-1p13-cholesterol-locus." class="section level3">
<h3>From noncoding variant to phenotype via SORT1 at the 1p13 cholesterol locus.</h3>
<p><em>Nature</em>.</p>
<p>N/A</p>
<p>2010</p>
<p>Musunuru K, Strong A, Frank-Kamenetsky M, Lee NE, Ahfeldt T, Sachs KV, Li X, Li H, Kuperwasser N, Ruda VM, Pirruccello JP, <strong>Muchmore B</strong>, Prokunina-Olsson L, Hall JL, Schadt EE, Morales CR, Lund-Katz S, Phillips MC, Wong J, Cantley W, Racie T, Ejebe KG, Orho-Melander M, Melander O, Koteliansky V, Fitzgerald K, Krauss RM, Cowan CA, Kathiresan S and Rader DJ.</p>
</div>
<div id="il28b-rs12979860-genotype-and-spontaneous-clearance-of-hepatitis-c-virus-in-a-multi-ethnic-cohort-of-injection-drug-users-evidence-for-a-supra-additive-association." class="section level3">
<h3>IL28B rs12979860 genotype and spontaneous clearance of hepatitis C virus in a multi-ethnic cohort of injection drug users: evidence for a supra-additive association.</h3>
<p><em>J. Infect. Dis</em>.</p>
<p>N/A</p>
<p>2011</p>
<p>F. M. Shebl, R. M. Pfeiffer, D. Buckett, <strong>B. Muchmore</strong>, S. Chen, M. Dotrang, L. Prokunina-Olsson, B. R. Edlin and T. R. O’Brien.</p>
</div>
<div id="common-genetic-variants-in-the-psca-gene-influence-gene-expression-and-bladder-cancer-risk." class="section level3">
<h3>Common genetic variants in the PSCA gene influence gene expression and bladder cancer risk.</h3>
<p><em>Proc. Natl. Acad. Sci. U.S.A.</em></p>
<p>N/A</p>
<p>2012</p>
<p>Y. P. Fu, I. Kohaar, N. Rothman, J. Earl, J. D. Figueroa, Y. Ye, N. Malats, W. Tang, L. Liu, M. Garcia-Closas, <strong>B. Muchmore</strong>, N. Chatterjee, M. Tarway, M. Kogevinas, P. Porter-Gill, D. Baris, A. Mumy, D. Albanes, M. P. Purdue, A. Hutchinson, A. Carrato, A. Tardon, C. Serra, R. Garcia-Closas, J. Lloreta, A. Johnson, M. Schwenn, M. R. Karagas, A. Schned, W. R. Diver, S. M. Gapstur, M. J. Thun, J. Virtamo, S. J. Chanock, J. F. Fraumeni, D. T. Silverman, X. Wu, F. X. Real and L. Prokunina-Olsson.</p>
</div>
<div id="il-29-is-the-dominant-type-iii-interferon-produced-by-hepatocytes-during-acute-hepatitis-c-virus-infection." class="section level3">
<h3>IL-29 is the dominant type III interferon produced by hepatocytes during acute hepatitis C virus infection.</h3>
<p><em>Hepatology.</em></p>
<p>N/A</p>
<p>2012</p>
<p>H. Park, E. Serti, O. Eke, <strong>B. Muchmore</strong>, L. Prokunina-Olsson, S. Capone, A. Folgori and B. Rehermann.</p>
<p class="spoiler">
Push Text Down
</p>
</div>
<div id="a-variant-upstream-of-ifnl3-il28b-creating-a-new-interferon-gene-ifnl4-is-associated-with-impaired-clearance-of-hepatitis-c-virus." class="section level3">
<h3>A variant upstream of IFNL3 (IL28B) creating a new interferon gene IFNL4 is associated with impaired clearance of hepatitis C virus.</h3>
<p><em>Nat. Genet.</em></p>
<p>N/A</p>
<p>2013</p>
<p>L. Prokunina-Olsson, <strong>B. Muchmore</strong>, W. Tang, R. M. Pfeiffer, H. Park, H. Dickensheets, D. Hergott, P. Porter-Gill, A. Mumy, I. Kohaar, S. Chen, N. Brand, M. Tarway, L. Liu, F. Sheikh, J. Astemborski, H. L. Bonkovsky, B. R. Edlin, C. D. Howell, T. R. Morgan, D. L. Thomas, B. Rehermann, R. P. Donnelly and T. R. O’Brien.</p>
</div>
<div id="the-19q12-bladder-cancer-gwas-signal-association-with-cyclin-e-function-and-aggressive-disease." class="section level3">
<h3>The 19q12 bladder cancer GWAS signal: association with cyclin E function and aggressive disease.</h3>
<p><em>Cancer Res.</em></p>
<p>N/A</p>
<p>2014</p>
<p>Y. P. Fu, I. Kohaar, L. E. Moore, P. Lenz, J. D. Figueroa, W. Tang, P. Porter-Gill, N. Chatterjee, A. Scott-Johnson, M. Garcia-Closas, <strong>B. Muchmore</strong>, D. Baris, A. Paquin, K. Ylaya, M. Schwenn, A. B. Apolo, M. R. Karagas, M. Tarway, A. Johnson, A. Mumy, A. Schned, L. Guedez, M. A. Jones, M. Kida, G. M. Hosain, N. Malats, M. Kogevinas, A. Tardon, C. Serra, A. Carrato, R. Garcia-Closas, J. Lloreta, X. Wu, M. Purdue, G. L. Andriole, R. L. Grubb, A. Black, M. T. Landi, N. E. Caporaso, P. Vineis, A. Siddiq, H. B. Bueno-de-Mesquita, D. Trichopoulos, B. Ljungberg, G. Severi, E. Weiderpass, V. Krogh, M. Dorronsoro, R. C. Travis, A. Tj?nneland, P. Brennan, J. Chang-Claude, E. Riboli, J. Prescott, C. Chen, I. De Vivo, E. Govannucci, D. Hunter, P. Kraft, S. Lindstrom, S. M. Gapstur, E. J. Jacobs, W. R. Diver, D. Albanes, S. J. Weinstein, J. Virtamo, C. Kooperberg, C. Hohensee, R. J. Rodabough, V. K. Cortessis, D. V. Conti, M. Gago-Dominguez, M. C. Stern, M. C. Pike, D. Van Den Berg, J. M. Yuan, C. A. Haiman, O. Cussenot, G. Cancel-Tassin, M. Roupret, E. Comperat, S. Porru, A. Carta, S. Pavanello, C. Arici, G. Mastrangelo, H. B. Grossman, Z. Wang, X. Deng, C. C. Chung, A. Hutchinson, L. Burdette, W. Wheeler, J. Fraumeni, S. J. Chanock, S. M. Hewitt, D. T. Silverman, N. Rothman and L. Prokunina-Olsson.</p>
</div>
<div id="cymer-cytometry-analysis-using-knime-docker-and-r." class="section level3">
<h3>CymeR: cytometry analysis using KNIME, docker and R.</h3>
<p><em>Bioinformatics.</em></p>
<p>N/A</p>
<p>2017</p>
<p><strong>B. Muchmore</strong> and M. E. Alarcon-Riquelme.</p>
</div>
<div id="molecular-characterization-of-monocyte-subsets-reveals-specific-and-distinctive-molecular-signatures-associated-with-cardiovascular-disease-in-rheumatoid-arthritis." class="section level3">
<h3>Molecular Characterization of Monocyte Subsets Reveals Specific and Distinctive Molecular Signatures Associated With Cardiovascular Disease in Rheumatoid Arthritis.</h3>
<p><em>Front Immunol.</em></p>
<p>N/A</p>
<p>2019</p>
<p>Ruiz-Limon P, Ortega-Castro R, Barbarroja N, Perez-Sanchez C, Jamin C, Patiño-Trives AM, Luque-Tevar M, Ibáñez-Costa A, Perez-Sanchez L, de la Rosa IA, Abalos-Aguilera M, Jimenez-Gomez Y, Calvo-Gutierrez J, Font P, Escudero-Contreras A, Alarcon-Riquelme ME, Collantes-Estevez E, López-Pedrera C; PRECISESADS Clinical Consortium and Flow Cytometry Study Group: Marañón C, Le Lann L, Varela N, <strong>Muchmore B</strong>, Dufour A, Alvarez EY, Carlo Montserrat Chizzolini C, De Langhe EN, Cl-P EY, Gerl V, De Groof A, Ducreux J, Trombetta E, Li T, Alvarez-Errico D, Rao S, Pers JO, Beretta L, AguilarQuesada R, Aguirre-Zamorano MA, Callejas Rubio JL, Castro-Villegas MC, Cervera R, Chizzolini C, Collantes E, Cornec D, De Langhe E, Devauchelle-Pensec V, Ae-C EY, Espinosa G, Fernández Roldán MC, Gomes Anjos T, Hiepe F, Jiménez Moleón I, Jousse-Joulin S, Lauwerys B, López-Berrio A, Lories R, Marovac J, Meroni PL, Miranda B, Navarro-Linares H, Ortega-Castro R, Ortego N, Pers JO, Ramón Garrido E, Raya E, Ríos Fernández R, Rodríguez-Pintó I, Saraux A.</p>
</div>
<div id="the-ifn-λ4-conundrum-when-a-good-interferon-goes-bad." class="section level3">
<h3>The IFN-λ4 Conundrum: When a Good Interferon Goes Bad.</h3>
<p><em>J. Interferon Cytokine Res.</em></p>
<p>N/A</p>
<p>2019</p>
<p>O. O. Onabajo, <strong>B. Muchmore</strong> and L. Prokunina-Olsson.</p>
<p class="spoiler">
Push Text Down
</p>
</div>
</div>
<div id="awards" class="section level2" data-icon="award">
<h2>Awards</h2>
<div id="license" class="section level3">
<h3>License</h3>
<p>Technology E-217-2011/1 titled “Identification of a Novel Interferon-Analog (IFNAN) Human Protein That Impairs Spontaneous and Treatment-Induced Clearance of Hepatitis C Virus in Humans”</p>
<p>United States Government</p>
<p>2012</p>
<p>Ludmila Prokunina-Olsson, Thomas R. O’Brien, <strong>Brian Muchmore</strong> and Raymond P. Donnelly.</p>
<p class="spoiler">
Push Text Down
</p>
</div>
<div id="patent" class="section level3">
<h3>Patent</h3>
<p>Novel Interferon-Lambda 4 (IFNL4) Protein, Related Nucleic Acid Molecules, and Uses Thereof.</p>
<p>United States Government</p>
<p>2012</p>
<p>Ludmila Prokunina-Olsson, Thomas R. O’Brien, <strong>Brian Muchmore</strong> and Raymond P. Donnelly.</p>
</div>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    // if there is no icon, add an icon:
    if (el.firstElementChild.firstChild && el.firstElementChild.firstChild.nodeName !== 'I') {
      s = el.dataset['icon'] || 'bookmark';
      el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
    }
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
